e2e/etcd_test_with_cov.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2017 The etcd Authors
```

Comment:
will change! What's a better name for `etcd_test_without_cov.go`?  `etcd_spawn.go`?